Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectBuilder/MSBuildProjectBuilder.cs

```diff
@@ -155,20 +157,28 @@ public static bool BuildProjects(this IReadOnlyCollection<MSBuildProjectReferenc
                     {
                         int completedProjects = 0;
                         string progressMessage = string.Empty;
+                        Match adoAuthenticationMatch = null;
                         var cancellationTokenSource = new CancellationTokenSource();
 
                         DisplayProgress();
 
                         Task<bool> buildTask = MSBuildProjectBuilder.BuildProjectsAsync(
                             msBuildProjectReferences,
-                            $"{arguments} -v:minimal",
+                            $"{arguments} -v:minimal -p:NuGetInteractive=true",
```

Comment:
>-p:NuGetInteractive=true [](start = 53, length = 24)

Does this argument work for a dotnet build as well? I would expect to have to switch to the --interactive argument for dotnet.

Reply:
Yes, this works with dotnet as well. The command ends up being: `dotnet msbuild Samples.sln -restore -t:Build -v:minimal -p:NuGetInteractive=true`.